core/cmd/local_client.go

```diff
@@ -339,13 +339,13 @@ func copyFile(src, dst string) error {
 	if err != nil {
 		return err
 	}
-	defer from.Close()
+	defer func() { _ = from.Close() }()
```

Comment:
Wow what is this voodoo?

Reply:
because it will tell unhandled error warning when doing static check if just type `from.Close()`